src/components/Body/Body.tsx

```diff
@@ -18,8 +18,8 @@ const Body: React.FC = () => {
       input: inputValue,
       selects: [selectedOption1, selectedOption2],
     });
-    setInputValue2(convertedValue); // TODO: set converted value in - setInputValue1
-    setInputValue1(inputValue); // TODO: copy velue in setInputValue2
+    setInputValue2(convertedValue); // TODO: set converted value in - setInputValue2
+    setInputValue1(inputValue); // TODO: copy velue in setInputValue1
```

Comment:
```suggestion
    setInputValue2(convertedValue); // set converted value in - setInputValue2
    setInputValue1(inputValue); // copy value in setInputValue1
```

Reply:
`TODO:` rather use for features or fixes, you have to get back later. For regular comment, rather use double slashes `// `